frontend/packages/rhoas-plugin/src/topology/createConnector.ts

```diff
@@ -0,0 +1,22 @@
+import { createServiceBinding } from "@console/topology/src/operators/actions/serviceBindings";
+import { Node } from '@patternfly/react-topology';
+
+export const getCreateConnector = (createHints: string[]) => {
+  if (
+    createHints
```

Comment:
This is disabled because I could not find reliable way to find hints. 
SBO integration works but generally arrow is not drawn for this

Reply:
Rather than using the default `nodeDropTargetSpec` you need to create your own and set the dropHint there.
See `OperatorBackedServcie.tsx`

Reply:
SBRs are not showing up because they are only crated by the operators plugin which only looks for connections to the OBS groups. You will need to do something similar in your data model transformer. See `operators-data-transformer.ts` for an example.

Reply:
I have done this but ended up having to move so many things that really made 2 places in the console that define SBO based connections.